thriftpy2/protocol/__init__.py

```diff
@@ -23,5 +24,6 @@
 __all__ = ['TBinaryProtocol', 'TBinaryProtocolFactory',
            'TCyBinaryProtocol', 'TCyBinaryProtocolFactory',
            'TJSONProtocol', 'TJSONProtocolFactory',
+           'TApacheJSONProtocol', 'TApacheJSONProtocolFactory',
            'TMultiplexedProtocol', 'TMultiplexedProtocolFactory',
-           'TCompactProtocol', 'TCompactProtocolFactory']
+           'TCompactProtocol', 'TCompactProtocolFactory', ]
```

Comment:
Maybe it is better to leave it as before.

Reply:
Fixed